saleor/webhook/subscription_payload.py

```diff
@@ -0,0 +1,116 @@
+from typing import Any, Dict, Optional
+
+from celery.utils.log import get_task_logger
+from django.core.handlers.base import BaseHandler
+from django.http import HttpRequest
+from django.test.client import RequestFactory
+from graphene_django.settings import graphene_settings
+from graphene_django.views import instantiate_middleware
+from graphql import get_default_backend, parse
+from graphql.error import GraphQLSyntaxError
+from graphql.execution.middleware import MiddlewareManager
+from promise import Promise
+
+from ..app.models import App
+
+logger = get_task_logger(__name__)
+
+
+def validate_subscription_query(query: str) -> bool:
+    from ..graphql.api import schema
+
+    graphql_backend = get_default_backend()
+    try:
+        graphql_backend.document_from_string(schema, query)
+    except (ValueError, GraphQLSyntaxError):
+        return False
+    return True
```

Comment:
We should also validate that the query contains subscriptions only.

Reply:
additional validation and tests added.